src/pilot/templates/nic-configs/4_port/nic_environment.yaml

```diff
@@ -45,3 +45,13 @@ parameter_defaults:
   StorageBond1Interface2: p4p2
   # The bonding mode to use for storage nodes
   StorageBondInterfaceOptions: mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
+
+  # CHANGEME: Change the interface names in the following lines for the
```

Comment:
I think we should create a new nic-config for this instead of trying to add to an existing NIC config.

Reply:
Agree. new nic-config dpdk_4_port and dpdk_5_port have been created.